bnd.bnd

```diff
@@ -0,0 +1,7 @@
+Bundle-Name: ${project.name}
+Bundle-Version: ${project.version}
+Bundle-SymbolicName: ${project.groupId}.${project.artifactId}
+Bundle-DocURL: ${project.url}
+
+Export-Package: org.example
+Import-Package: *
```

Comment:
This line could be removed as the value * is the default of _Import-Package_.  
But for the sake of completeness it can of course be kept.

Reply:
I try to favor expressiveness over hidden defaults and would leave it for now.